config/fcos/v1_5_exp/translate.go

```diff
@@ -78,6 +80,14 @@ func (c Config) ToIgn3_4Unvalidated(options common.TranslateOptions) (types.Conf
 			}
 		}
 	}
+
+	retp, rp, tsp := c.processPackages(options)
+
+	ret.Storage.Files = append(ret.Storage.Files, retp.Storage.Files...)
```

Comment:
You can't just glue some arrays together here; you need to run a proper config merge with `MergeTranslatedConfigs`.  That'll take care of mapping the translation set indices for you.